com.unity.render-pipelines.universal/package.json

```diff
@@ -6,7 +6,8 @@
   "unityRelease": "0a17",
   "displayName": "Universal RP",
   "dependencies": {
-    "com.unity.mathematics": "1.1.0",
+    "com.unity.mathematics": "1.2.1",
+    "com.unity.burst": "1.5.0",
```

Comment:
My only concern about adding burst, in decals we seen mobile crashing in some scenarios. I fear it might no be stable enough